README.md

```diff
@@ -39,18 +39,17 @@ The Chrome driver is available at https://chromedriver.chromium.org/
 
 ## Running ZAP scan locally
 
-To run the ZAP scan, you will need a copy of the ZAP proxy running locally: https://www.zaproxy.org/, with the 
-following options configured:
+To run the ZAP scan, use the Docker helper supplied by `dast-config-manager` (https://github.com/hmrc/dast-config-manager#running-zap-locally)
 
-* port configured to 11000
-* under HUD, uncheck 'Enable when using the ZAP Desktop' (stops ZAP converting requests to HTTPS)
-* under API, check 'Disable the API key'
+Follow the following steps:
+1. Clone the repo at: https://github.com/hmrc/dast-config-manager
+2. Enable port forwarding: `export ZAP_FORWARD_ENABLE="true"`
```

Comment:
Should we aim to move this logic into a script, repurposing the existing run_zap_tests.sh?